javalib/src/main/scala/java/util/HashSet.scala

```diff
@@ -24,7 +26,7 @@ class HashSet[E] extends AbstractSet[E] with Set[E]
     inner.contains(Box(o.asInstanceOf[E]))
 
   override def remove(o: Any): Boolean =
-    inner.remove(Box(o.asInstanceOf[E]))
+    inner.remove(Box(o.asInstanceOf[E])).nonEmpty
```

Comment:
The `remove` operation now returns an `Option` on `mutable.Set`. See also https://github.com/scala/collection-strawman/issues/551